curriculum/challenges/english/25-front-end-development/lecture-html-fundamentals/6708143cab2b583ecd3324f5.md

```diff
@@ -2,7 +2,7 @@
 id: 6708143cab2b583ecd3324f5
 title: What Are Attributes, and How Do They Work?
 challengeType: 11
-videoId: nVAaxZ34khk
+videoId: V9dOuY5BdDo
```

Comment:
It seems like this video id changed at some point. Here's the current id:

```suggestion
videoId: zUCIsEDCIMU
```

Reply:
thanks Kris 👍

Reply:
thanks, looks like I just forgot to commit the suggestion 😬